### ISSUE TITLE
Could more gracefully handle error

### ISSUE THREAD

**Issue by ryancwalsh** [sampotts/rangetouch#5]
I just found out that the way I was using Rangetouch broke my site for about a week.

I guess I accidentally moved `require("rangetouch/dist/rangetouch");` to *before* `$(document).ready(function () {`.

And I didn't have it within a `try/catch`.

What would have been nice is if Rangetouch wouldn't cause an exception (causing all my other javascript to fail) just because at the time of it being called there was no input range rendered on the page.

Thanks so much for your cool tool!

I'm now successfully using it like this:

```
$(document).ready(function () {
        try {
            require("rangetouch/dist/rangetouch");//This must be called *after* $(document).ready() https://rangetouch.com/ https://github.com/sampotts/rangetouch
        } catch (e) {
            console.log(e);
        }
...
```

**Comment by sampotts** [comment]
Sorry about that. I'll get this fixed up today given it will be a tiny fix...

**Comment by sampotts** [comment]
Hey,

This should be resolved in [v1.0.3](https://github.com/sampotts/rangetouch/releases/tag/v1.0.3)

**Comment by ryancwalsh** [comment]
@sampotts Awesome! Thanks for your hard work!
